node/pkg/reporter/app.go

```diff
@@ -78,78 +78,46 @@ func (a *App) setReporters(ctx context.Context) error {
 			WithInterval(groupInterval),
 			WithContractAddress(contractAddress),
 			WithCachedWhitelist(cachedWhitelist),
+			WithKaiaHelper(chainHelper),
+			WithLatestData(&a.LatestData),
```

Comment:
let's define app.LatestData as *sync.Map type, and allocate memory from App.New() func. then, we can remove `&` here

Reply:
done! will be reflected in the next PR